client/src/components/Console.tsx

```diff
@@ -95,8 +91,18 @@ export default function Console() {
       fetchDocuments();
       setRefreshNeeded(false);
     }
-  }, [selectedView, refreshNeeded]);
+  }, [refreshNeeded]);
+  useMemo(() => {
+    const entireMunicipalityDocs = tmpDocuments.filter(
+      (doc) => doc.doc_coordinates?.type === "EntireMunicipality"
+    );
+    const otherDocs = tmpDocuments.filter(
```

Comment:
https://github.com/user-attachments/assets/17add999-9c01-4e1a-8f21-879c102d3a4b

after working with filtering a while, I see this condition!

Reply:
unfortunately i did not manage to reproduce it